test_fms/fft/test_fft.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-cp $srcdir/test_fms/fft/input.nml input.nml
```

Comment:
Note, there was no file called input.nml. It was already names input_base.nml, so this copy command must be failing in your test, when lead me to realize you need to do a set -e in every test script.

Reply:
Seems like input.nml was created by the test, otherwise your make distcheck could not have worked...

Reply:
OK, never mind. I think I was looking at artifacts of a previous attempt in this directory. Travis straightened me out.